zulipterminal/ui_tools/views.py

```diff
@@ -1023,7 +1026,28 @@ def __init__(self, controller: Any, msg: Message, title: str) -> None:
                     grouped_reactions[reaction] = user
             msg_info.append(('Reactions', list(grouped_reactions.items())))
 
+        # Render the category using the existing table methods if links exist.
+        if message_links:
+            msg_info.append(('Message Links', []))
+
         popup_width, column_widths = self.calculate_table_widths(msg_info,
                                                                  len(title))
         widgets = self.make_table_with_categories(msg_info, column_widths)
+
+        if message_links:
+            message_link_widgets = []
+            message_link_width = 0
+            for index, link in enumerate(message_links):
+                text, link_index, _ = message_links[link]
+                caption = '{}: {}\n{}'.format(index, text, link)
+                message_link_width = max(message_link_width, len(caption))
+
+                display_attr = None if index % 2 else 'popup_contrast'
+                message_link_widgets.append(
+                    urwid.AttrMap(urwid.Text(caption), display_attr)
+                )
+
+            widgets.extend(message_link_widgets)
```

Comment:
I am currently showing `message_links` at the bottom since it is straightforward to implement and we were planning to move reactions away from `MsgInfoView` in #707. Though, I am not sure when that would be integrated. Thoughts?

Reply:
Resolved in PM. (We're moving ahead with `message_links` before 'Reactions'.)